client/network/src/config.rs

```diff
@@ -643,6 +731,88 @@ impl NonReservedPeerMode {
 	}
 }
 
+///Doc
+#[derive(Clone, Debug)]
+pub enum PreShareKeyConfig {
+	/// A pre-shared key configuration.
+	PRESHAREDKEY(PreSharedKeySecret),
+}
+
+/// The options for obtaining a pre-shared key.
+pub type PSKey = PreSharedKeySecret;
+
+///Doc
+#[derive(Clone)]
+pub enum PreSharedKeySecret {
+	/// Read the pre-shared key from a file.
+	File(PathBuf),
+}
+
+impl Default for PreShareKeyConfig {
+	fn default() -> PreShareKeyConfig {
+		let path = PathBuf::from("not-used");
```

Comment:
Where is this file gonna be created? Shouldn't we create it in "tmp" folder? Btw. this is not used at the moment, if you're not gonna use it - remove

Reply:
ok